lib/services/api_connect.dart

```diff
@@ -282,18 +189,18 @@ class APIConnect {
     };
     var response = (httpReqType == 'POST') // need pid, uid, and voteVal
         ? await http.post(
-            "$_url/votes",
+            "$url/votes",
             body: json.encode(vote),
             headers: await headers,
           )
         : (httpReqType == 'GET') // need pid and uidUser
             ? await http.get(
-                "$_url/votes/$pid/$uidUser",
+                "$url/votes/$pid/$uidUser",
                 headers: await headers,
               )
             : await http.patch(
                 // need pid, uid, and voteVal
-                "$_url/votes",
+                "$url/votes",
                 body: json.encode(vote),
                 headers: await headers,
               );
```

Comment:
change this into if/else-if statements

Reply:
why

Reply:
cuz cleaner and there are no nested ternaries

Reply:
wouldnt the if statements also be nested

Reply:
var response;
if (httpReqType == 'POST') {
  await http.post(
            "$_url/votes",
            "$url/votes",
            body: json.encode(vote),
            headers: await headers,
  );
} else if (httpReqType == 'GET') {
  await http.get(
                "$_url/votes/$pid/$uidUser",
                "$url/votes/$pid/$uidUser",
                headers: await headers,
  );
} else if (httpReqType == 'PATCH') {
  await http.patch(
                // need pid, uid, and voteVal
                "$_url/votes",
                "$url/votes",
                body: json.encode(vote),
                headers: await headers,
              );
}